### ISSUE TITLE
Simple Onboarding

### ISSUE THREAD

**Issue by COM8** [UWPX/UWPX-Client#60]
Make it easier to onboard people that do non know anything about XMPP.

#### References:
* [Easy XMPP Invitation Landing Page]()
* [Modern XMPP](https://docs.modernxmpp.org/)

**Comment by Echolon** [comment]
Also take a look at my suggestions here:

[UI/UX] Add welcome screen: User should be invited to register a new or existing account
https://github.com/anurodhp/Monal/issues/128#issuecomment-479859655

FR: Tutorial for newbies https://github.com/kriztan/Pix-Art-Messenger/issues/172

This is also important as we usually have the problem that they don't have friend. It's always easier for them to quickly and simply add new one (which I mostly send them after they added me):

Inline add button for XMPP-URIs (contact or MUC)
https://github.com/anurodhp/Monal/issues/217

I always recommened to build a system setup a long a principle which I used to call **"Fire & Forget"**. This idea about consistent onboarding fulfills to being complete once a new user gets an invitation and just sends me his or her ID and we are ready to chat. Maybe even not the last action (because it maybe can be somehow automised or linked). 
So the app will guide and introduce the user to XMPP, to the client as well as help them to easily onboard without hassle as well as haveing default settings well configured. This counts also to the default server for registration!

**Comment by COM8** [comment]
Yes, was planning to do something like the by you mentioned "Fire and Forget" in Berlin with the possibility for the user to creat accounts just by using the app.
